packages/app/src/components/search/SearchResultCustomList.tsx

```diff
@@ -7,12 +7,13 @@ import { CatalogSearchResultListItem } from '@backstage/plugin-catalog';
 import { StackOverflowSearchResultListItem, StackOverflowIcon } from '@backstage/plugin-stack-overflow';
 import { CatalogIcon, DocsIcon } from '@backstage/core-components';
 import { TechDocsSearchResultCustomListItem } from './TechDocsSearchResultCustomListItem';
+import { refreshLinkClickTracking } from '@snowplow/browser-plugin-link-click-tracking';
 
 const SearchResultCustomList = () => {
     return (
         <SearchResult>
             {({ results }) => (
-                <List>
+                <List onMouseOver={() => refreshLinkClickTracking()}>
```

Comment:
Wondering why mouse over is used vs a click event? Is it so there is the tracking refresh on any movement? Would that skew the results if the user moves the mouse a lot?

Reply:
Hi @MonicaG 
I've tested using the click event, and the first link click isn't captured (subsequent link clicks are caught).
The results shouldn't be skewed since "refreshLinkClickTracking" doesn't generate an event itself, rather it will add click event listeners to all links.

After reviewing this code I changed the "onMouseOver" event to "onMouseEnter" since the former will trigger via child elements which is not useful behaviour.